manager/state/raft/raft_test.go

```diff
@@ -327,74 +327,6 @@ func TestRaftFollowerLeave(t *testing.T) {
 	assert.Len(t, nodes[4].GetMemberlist(), 4)
 }
 
-func TestRaftLeaderLeave(t *testing.T) {
```

Comment:
Should this test be modified to just fail with the expected error rather than removed entirely?